### DISCUSSION TITLE
About UI language

### DISCUSSION THREAD

**Discussion by swordpp** [Stirling-Tools/Stirling-PDF#1722] [Q&A] [answered]
I created a Docker container, using parameters -e LANGS=zh_TW.
Then I opened the web interface, it displays the English interface, and i set the interface language, but the next time I open it, it still displays in English. Why?
I am using command:
docker run -d \
  -p 8080:8080 \
  -v /home/s-pdf/trainingData:/usr/share/tessdata \
  -v /home/s-pdf/extraConfigs:/configs \
  -v /home/s-pdf/logs:/logs \
  -v /home/s-pdf/customFiles:/customFiles \
  -e DOCKER_ENABLE_SECURITY=false \
  -e INSTALL_BOOK_AND_ADVANCED_HTML_OPS=false \
  -e LANGS=zh_TW \
  --name s-pdf \
  frooodle/s-pdf:0.27.0-ultra-lite

**Comment by Frooodle** [comment, marked as answer]
```LANGS to define custom font libraries to install for use for document conversions```

What you want is 
``SYSTEM_DEFAULTLOCALE: en_US``
